src/bci_build/package/__init__.py

```diff
@@ -846,6 +860,34 @@ def packages(self) -> str:
                 )
         return " ".join(str(pkg) for pkg in self.package_list)
 
+    @property
+    def pretty_extra_packages(self) -> list[str]:
+        """Returns the non-core packages in this image. It is intended
+        to be used in the image documentation to create an additional
+        packages section.
+
+        """
+        package_list = []
+
+        for pkg in self.package_list:
+            if isinstance(pkg, Package):
+                if pkg.pkg_type != PackageType.IMAGE:
+                    continue
+
+                if pkg.pkg_category == PackageCategory.CORE:
+                    continue
+
+                name = pkg.name
+            else:
+                name = pkg
+
+            if name in self.os_version.lifecycle_data_pkg:
+                continue
+
+            package_list.append(name)
+
+        return sorted(package_list)
```

Comment:
I think that the extra packages should be an opt-in feature and not opt-out. With the current behavior, `pretty_extra_packages` is equal to `package_list` for all containers built with Docker/Podman. And for those using kiwi + bootstrap packages, it'll be empty. I'd rather try this:
```suggestion
    @property
    def pretty_extra_packages(self) -> list[str]:
        """Returns the non-core packages in this image. It is intended
        to be used in the image documentation to create an additional
        packages section.

        """
        return sorted([pkg.name for pkg in self.package_list if isinstance(pkg, Package) and pkg.pkg_category == PackageCategory.CORE])
```

Reply:
Yes, that is something I was considering. I believe if we make exclusive `PackageCategory.EXTRA` it is a lot clearer what this function will output to a README.

You wrote `PackageCategory.CORE`, I assume you intended to write `PackageCategory.EXTRA` based on your comment.